specification/loadtestservice/data-plane/Microsoft.LoadTestService/stable/2022-11-01/loadtestservice.json

```diff
@@ -0,0 +1,2958 @@
+          "Lists the metric values for a load test run.": {
+            "$ref": "./examples/GetTestRunMetrics.json"
+          },
+          "Lists the metric metadata for a load test run.": {
+            "$ref": "./examples/GetTestRunMetricsMetadata.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AppComponent": {
+      "description": "An Azure resource object (Refer azure generic resource model : https://docs.microsoft.com/en-us/rest/api/resources/resources/get-by-id#genericresource)",
+      "required": [
+        "resourceId",
+        "resourceName"
+      ],
+      "type": "object",
+      "properties": {
+        "resourceId": {
+          "description": "fully qualified resource Id e.g subscriptions/{subId}/resourceGroups/{rg}/providers/Microsoft.LoadTestService/loadtests/{resName}",
+          "type": "string"
+        },
+        "resourceName": {
+          "description": "Azure resource name, required while creating the app component.",
+          "type": "string"
+        },
+        "resourceType": {
+          "description": "Azure resource type, required while creating the app component.",
+          "type": "string"
+        },
+        "displayName": {
+          "description": "Azure resource display name",
+          "type": "string"
+        },
+        "resourceGroup": {
+          "description": "Resource group name of the Azure resource",
+          "type": "string",
+          "readOnly": true
+        },
+        "subscriptionId": {
+          "description": "Subscription Id of the Azure resource",
+          "type": "string",
+          "readOnly": true
+        },
+        "kind": {
+          "description": "Kind of Azure resource type",
+          "type": "string"
+        }
+      }
+    },
+    "AppComponentsMap": {
+      "description": "App Components model",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "resourceId": {
+          "description": "Azure Load Testing resource Id",
+          "type": "string",
+          "readOnly": true
+        },
+        "testId": {
+          "description": "[Required, if testRunId is not given] Load test unique name",
+          "type": "string"
+        },
+        "testRunId": {
+          "description": "[Required if testId is not given] Load test run unique name",
+          "type": "string"
+        },
+        "name": {
+          "description": "AppComponent name",
+          "type": "string",
+          "readOnly": true
+        },
+        "value": {
+          "description": "AppComponents collection { resource id (fully qualified resource Id e.g subscriptions/{subId}/resourceGroups/{rg}/providers/Microsoft.LoadTestService/loadtests/{resName}) : resource object } ",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/AppComponent"
+          }
+        }
+      }
+    },
+    "CertificateMetadata20221101": {
```

Comment:
Please remove the "20221101" suffix from everything.

Reply:
removed